mlrun/datastore/targets.py

```diff
@@ -1413,6 +1413,27 @@ def as_df(self, columns=None, df_module=None, **kwargs):
 
 
 class KafkaTarget(BaseStoreTarget):
+    """Kafka target storage driver, used to materialize feature set
+     data into kafka topics.
+
+    example::
+
+        # define target
+        kafka_target = KafkaTarget(
+            "kafka",
+            path='my_topic',
+            bootstrap_servers='localhost:9092'
+        )
+
+        # ingest
+        stocks_set.ingest(stocks, [kafka_target])
+
+    :param name:        optional, target name.
+    :param path:        optional, topic name e.g. 'my_topic' or 'v3io:///{topic}'.
```

Comment:
It's not really optional though, despite being technically optional in the constructor.